schedule/utils.py

```diff
@@ -183,11 +187,11 @@ def decorator(request, *args, **kwargs):
         return HttpResponseNotFound('<h1>Page not found</h1>')
     return decorator
 
+
 def check_calendar_permissions(function):
     @wraps(function)
     def decorator(request, *args, **kwargs):
         if CALENDAR_VIEW_PERM:
-            from schedule.models import Event, Calendar, Occurrence
```

Comment:
I think this change could bring some recursive issues that we had in the past.
I agree we need to move this import from here but I remember an issue with this.
could you revert this? (unless you are sure this will not break something)